pkg/controller/remotemachineset/ovirt_test.go

```diff
@@ -39,7 +39,7 @@ func TestOvirtActuator(t *testing.T) {
 			clusterDeployment: testOvirtClusterDeployment(),
 			pool:              testOvirtPool(),
 			expectedMachineSetReplicas: map[string]int64{
-				fmt.Sprintf("%s-worker-0", testInfraID): 3,
+				fmt.Sprintf("%s-worker", testInfraID): 3,
```

Comment:
This is worrisome.

Reply:
just so i understand , previously they had -0 suffix even when there was only on machineset for a machinepoolm, but they have replaced the name to no suffix. and our worry is that it will affect our existing clusters as they will now have a new machineset and abandon the last one?

Reply:
Correct. My worry is that if the naming changes we will replace machinesets for existing ovirt clusters.

Reply:
Added a function to preserve the suffix.